### PR TITLE
Mention support launchpad. Minor rewrite for clarity and style.

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Bring more clarity behind our support channels and process.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
~~- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.~~

### CODE REVIEW

File: docs/overview/support.mdx [hunk 1]
```diff
@@ -12,29 +12,46 @@ keywords:
...
-## Reporting a BCP Incident for Registered SAP Customers
+## About Support
 
-- As a registered SAP customer, report issues by creating incidents in [SAP Support Portal](http://help.sap.com/disclaimer?site=https://support.sap.com/home.html).
-- Use **XX-S4C-SDK** component to reach out to us.
+At the SAP Cloud SDK we do our best to provide best in class support experience.
+We monitor key support metrics and make sure no issues are left unanswered in our official channels.
 
-### How Responsive Are You?
+### Response times
```
Comment (artemkovalyov) [inline docs/overview/support.mdx:49]:
```suggestion
### Response Time
```

File: docs/overview/support.mdx [hunk 2]
```diff
@@ -12,29 +12,46 @@ keywords:
...
 
-- You will usually hear from us within 1-2 days.
+- We reply within **1-2 business days** to the majority of the issues.
```
Comment (artemkovalyov) [inline docs/overview/support.mdx:51]:
```suggestion
- We operate during the working hours of the Berlin CET time zone.
- We reply within **1-2 business days** to the majority of the issues.
```

File: docs/overview/support.mdx [hunk 3]
```diff
@@ -12,29 +12,47 @@ keywords:
...
 
 ### Stack Overflow
 
-- Use the `sap-cloud-sdk` tag on [Stack Overflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk)
+Use the `sap-cloud-sdk` tag on [Stack Overflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk) to search for solved issues or create yours.
+
+## For Registered SAP Customers - Use SAP One Support Launchpad
+
+As a registered SAP customer you can use the [SAP One Support Launchpad](https://launchpad.support.sap.com/) to report issues.
+We recommend using this channel when you need to share **internal or sensitive** information. In other cases [public support channels](#public-support-channels) are preferred.
```
Comment (artemkovalyov) [inline docs/overview/support.mdx:36]:
```suggestion
We recommend using this channel when you need to share **internal or sensitive** information.
In other cases [public support channels](#public-support-channels) are preferred.
```

File: docs/overview/support.mdx [hunk 4]
```diff
@@ -12,29 +12,48 @@ keywords:
...
+
+:::
 
-## Reporting a BCP Incident for Registered SAP Customers
+## About Support
 
-- As a registered SAP customer, report issues by creating incidents in [SAP Support Portal](http://help.sap.com/disclaimer?site=https://support.sap.com/home.html).
-- Use **XX-S4C-SDK** component to reach out to us.
+At the SAP Cloud SDK we do our best to provide best in class support experience.
+We monitor key support metrics and make sure no issues are left unanswered in our official channels.
```
Comment (artemkovalyov) [inline docs/overview/support.mdx:48]:
```suggestion
At the SAP Cloud SDK, we do our best to provide best in class support experience.
We monitor key support metrics and make sure no issues are left unanswered in our official channels.
Our official support language is English.
```

File: docs/overview/support.mdx [hunk 5]
```diff
@@ -12,29 +12,49 @@ keywords:
...
+Our official support language is English.
 
-### How Responsive Are You?
+### Response Time
 
-- You will usually hear from us within 1-2 days.
+- We operate during the working hours of the Berlin CET time zone.
+- We reply within **1-2 business days** to the majority of the issues.
+  Our median response time is **less than 1 day**.
```
Comment (MatKuhr) [inline docs/overview/support.mdx:55]: Not sure if these guarantees are fine to make. It's not a hard commitment I think.

**Review by MatKuhr — APPROVED** [review]
generally lgtm, maybe reconsider the response time commitments

File: docs/overview/support.mdx (hunk 5 above)
Comment (artemkovalyov) [reply docs/overview/support.mdx:55]: Yep, this is not a contract-type commitment. More of what to expect.

**Pull request merged**